<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SvFlash</class>
 <widget class="QDialog" name="SvFlash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SvFlash</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Состояние подключения:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>80</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Файл для загрузки:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mStatus">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>671</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="mFile">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>80</y>
     <width>671</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="mSelect">
   <property name="geometry">
    <rect>
     <x>521</x>
     <y>120</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать файл</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mFlash">
   <property name="geometry">
    <rect>
     <x>631</x>
     <y>120</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Прошить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mExit">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>120</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Состояние прошивки:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mFlashStatus">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>671</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
